src/appMain/sdl_preloaded_pt.json

```diff
@@ -3914,6 +3918,30 @@
                     "mandatory": false,
                     "since": "5.1"
                 },
+                {
+                    "name": "windowStatus",
+                    "params": [
+                        {
+                            "name": "location",
+                            "key": "OEM_REF_WND_LOC",
+                            "type": "Grid",
+                            "mandatory": true
+                        },
```

Comment:
Referring back to this comment https://github.com/smartdevicelink/sdl_core/pull/3304#discussion_r449029139. The type can only be an enum or a base type. Grid is a Struct so it should be expanded here.
```suggestion
                        {
                            "name": "location",
                            "params": [
                                {
                                    "name": "col",
                                    "key": "OEM_REF_WND_LOC_COL",
                                    "type": "Integer",
                                    "mandatory": true,
                                    "minvalue": -1,
                                    "maxvalue": 100
                                },
                                {
                                    "name": "row",
                                    "key": "OEM_REF_WND_LOC_ROW",
                                    "type": "Integer",
                                    "mandatory": true,
                                    "minvalue": -1,
                                    "maxvalue": 100
                                },
                                {
                                    "name": "level",
                                    "key": "OEM_REF_WND_LOC_LVL",
                                    "type": "Integer",
                                    "mandatory": false,
                                    "defvalue" : 0,
                                    "minvalue": -1,
                                    "maxvalue": 100
                                },
                                {
                                    "name": "colspan",
                                    "key": "OEM_REF_WND_LOC_COLSPN",
                                    "type": "Integer",
                                    "mandatory": false,
                                    "defvalue" : 1,
                                    "minvalue": 1,
                                    "maxvalue": 100
                                },
                                {
                                    "name": "rowspan",
                                    "key": "OEM_REF_WND_LOC_ROWSPN",
                                    "type": "Integer",
                                    "mandatory": false,
                                    "defvalue" : 1,
                                    "minvalue": 1,
                                    "maxvalue": 100
                                },
                                {
                                    "name": "levelspan",
                                    "key": "OEM_REF_WND_LOC_LVLSPN",
                                    "type": "Integer",
                                    "mandatory": false,
                                    "defvalue" : 1,
                                    "minvalue": 1,
                                    "maxvalue": 100
                                }
                            ],
                            "key": "OEM_REF_WND_LOC",
                            "type": "Struct",
                            "mandatory": true
                        },
```